app/server_inactivity.go

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package app
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+	"github.com/mattermost/mattermost-server/v6/shared/mlog"
+	"github.com/mattermost/mattermost-server/v6/store"
+)
+
+const serverInactivityHours = 100
+
+func (s *Server) doInactivityCheck() {
+	inactivityDurationHourseEnv := os.Getenv("MM_INACTIVITY_DURATION")
+	inactivityDurationHours, parseError := strconv.ParseFloat(inactivityDurationHourseEnv, 64)
+	if parseError != nil {
+		// default to 100 hours
+		inactivityDurationHours = serverInactivityHours
+	}
+
+	systemValue, sysValErr := s.Store.System().GetByName("INACTIVITY")
+	if sysValErr != nil {
+		if _, ok := sysValErr.(*store.ErrNotFound); !ok {
+			mlog.Error("An error occurred while getting INACTIVITY from system store", mlog.Err(sysValErr))
```

Comment:
Should we downgrade this to a warn or info given that is a non-essential check ?